test_scripts/Polices/Policy_Table_Update/014_ATF_P_TC_PTU_retry_timeout_definition.lua

```diff
@@ -0,0 +1,106 @@
+---------------------------------------------------------------------------------------------
+-- Requirements summary:
+-- PoliciesManager must use the values from "seconds_between_retries" section of 
```

Comment:
Please change to [PolicyTableUpdate] Policy Table Update retry timeout definition